app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextConverter.java

```diff
@@ -91,13 +94,10 @@
 
     public static final String HTML_KATEX_INCLUDE = "<link rel='stylesheet'  type='text/css' href='file:///android_asset/katex/katex.min.css'>" +
             "<script src='file:///android_asset/katex/katex.min.js'></script>" +
-            "<script src='file:///android_asset/katex/mhchem.min.js'></script>" +
-            "<script src='file:///android_asset/katex/auto-render.min.js'></script>";
-    public static final String JS_KATEX = "" +
-            "renderMathInElement(document.body, {" +
-            "   'delimiters': [ " +
-            "       {left: '$$', right: '$$', display: true}, { left: '$', right: '$', display: false }," +
-            "]});\n";
+            "<script src='file:///android_asset/katex/katex-render.js'></script>" +
+            "<script src='file:///android_asset/katex/mhchem.min.js'></script>";
+    public static final String CSS_KATEX = CSS_S + ".katex { font-size: 105%; }" + CSS_E;
```

Comment:
humble opinion: equations can get really looooong. I think we can live without custom font size rule :smile: .

Or does it make such a noticeable difference, and you think it's much required?